byway/src/main/webapp/Destinations.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAljWlTjyagvHyZTN69Mvun2mH4Eq8j9yI&callback=initAutocomplete&libraries=places&directions&v=weekly"
+      defer
+    ></script>
+    <link href="https://fonts.googleapis.com/css2?family=Karla&family=Rubik:wght@500;700&display=swap" rel="stylesheet">
+  </head>
+  <body id="destinations-page" onload="getLocations(); getStartDestination(); setProgressBar(1)" >
+
+    
+    <div class="header">
+      <img src="/images/logo.png" alt="logo" class="logo">
+      <div class="header-right">
+        <a>byway.</a>
```

Comment:
add a link since it's an a tag

Reply:
changed to div instead  idk why i had it as an a tag haha